app/grandchallenge/workstation_configs/models.py

```diff
@@ -270,6 +271,21 @@ class ImageInterpolationType(models.TextChoices):
         validators=[MinValueValidator(limit_value=0.01)],
     )
 
+    default_brush_size = models.DecimalField(
+        blank=True,
+        null=True,
+        max_digits=4,
+        decimal_places=3,
+        validators=[MinValueValidator(limit_value=0.001)],  # 1µm
```

Comment:
Is this OK for pathology?

Reply:
Good question, average length of a cell is 120um, so I thought I'd be enough to do 1um. However, mitochondria and other organoids in cells might require a finer touch. (Or viri particles/bacteria). I'll make it 1nm. That should def be clear on anything organic.